SensusService/Protocol.cs

```diff
@@ -387,6 +392,10 @@ public static void RunUnitTestingProtocol(Stream unitTestingProtocolFile)
         private bool _shareable;
         private List<PointOfInterest> _pointsOfInterest;
         private string _description;
+        private DateTime _startTime;
+        private bool _startImmediately;
+        private DateTime _endTime;
+        private bool _continueIndefinitely;
```

Comment:
Similarly, make the start and end `DateTime` fields nullable and interpret null values as immediate (for start) and indefinite (for end). Then remove the bools.